shells/lib/components/arc-host.js

```diff
@@ -15,12 +15,12 @@ import {devtoolsArcInspectorFactory} from '../../../build/devtools-connector/dev
 const {log, warn, error} = logsFactory('ArcHost', '#cade57');
 
 export class ArcHost {
-  constructor(context, storage, composer, storageservice, portFactories) {
+  constructor(context, storage, composer, storagemanager, portFactories) {
```

Comment:
storagemanager should probably be camel case here (storageManager).

Reply:
done. 
(but i didn't completely make it up, in arc-component.js the observed attributes actually do not support camel case, must all be lowercase).